src/main.rs

```diff
@@ -53,6 +54,9 @@ use x509::time::{Time, Validity};
 use x509::{Certificate, TbsCertificate};
 use zeroize::Zeroizing;
 
+const PKCS10: &str = "application/pkcs10";
+const PKCS10_BUNDLE: &str = "application/vnd.steward.pkcs10-bundle.v1";
```

Comment:
```suggestion
const BUNDLE: &str = "application/vnd.steward.pkcs10-bundle.v1";
```

The long name will cause line wraps in match statements.

Reply:
Done